internal/cloud/atlas/groups.go

```diff
@@ -7,8 +7,9 @@ import (
 	"github.com/10gen/realm-cli/internal/utils/api"
 )
 
+// GroupsPath is the route for group API handlers
 const (
-	groupsPath = publicAPI + "/groups"
+	GroupsPath = publicAPI + "/groups"
```

Comment:
would keep this unexported if its otherwise solely for testing purposes... the test can just use a raw string